hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcClient.java

```diff
@@ -997,6 +1008,20 @@ public <T> T invokeSequential(
         expectedResultValue).getResult();
   }
 
+  /**
+   * Same as the above invokeSequential, but with router ugi.
+   */
+  public <T> T invokeSequentialAsRouter(
```

Comment:
Nit Javadoc on public function.

Reply:
don't want to copy the whole same section from invokeSequential. We will keep it short here.